CONTRIBUTING.md

```diff
@@ -6,21 +6,22 @@ This guide outlines the process of landing patches in gocql and the general appr
 
 ## Background
 
-The goal of the gocql project is to provide a stable and robust CQL driver for Go. gocql is a community driven project that is coordinated by a small team of core developers.
+The goal of the gocql project is to provide a stable and robust CQL driver for Go.  This is a community driven project that is coordinated by a small team of developers in and around the Apache Cassandra project.  For security, governance and administration issues please refer to the Cassandra Project Management Committee.
 
 ## Minimum Requirement Checklist
 
 The following is a check list of requirements that need to be satisfied in order for us to merge your patch:
 
-* You should raise a pull request to gocql/gocql on Github
+* You should raise a pull request to apache/cassandra-gocql-driver on Github
 * The pull request has a title that clearly summarizes the purpose of the patch
 * The motivation behind the patch is clearly defined in the pull request summary
-* Your name and email have been added to the `AUTHORS` file (for copyright purposes)
+* You agree that your contribution is donated to the Apache Software Foundation (appropriate copyright is on all new files)
 * The patch will merge cleanly
 * The test coverage does not fall below the critical threshold (currently 64%) 
 * The merge commit passes the regression test suite on Travis
 * `go fmt` has been applied to the submitted code
 * Notable changes (i.e. new features or changed behavior, bugfixes) are appropriately documented in CHANGELOG.md, functional changes also in godoc
+* A correctly formatted commit message, see https://cassandra.apache.org/_/development/how_to_commit.html#tips
```

Comment:
It seems to me that it would be helpful to expand the guidance on writing commit messages to a separate section.

Specifically, to say that the description of the commit should explain why the change is necessary

Should we encourage contributors to use signed git commits?

Reply:
> Should we encourage contributors to use signed git commits?

This isn't project precedence, though it can always be recommended (and should be).
Ultimately, it's not my call.